backend/vcdat/app.py

```diff
@@ -1,9 +1,22 @@
-from flask import Flask
-app = Flask(__name__)
+from flask import Flask, send_from_directory
+import os
+app = Flask(__name__, static_url_path='')
 
 @app.route("/")
 def hello():
-    return "Hello, Climate"
+    path = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', 'frontend/src/'))
+    return send_from_directory(path, 'index.html')
+
+@app.route("/deps/<path:path>")
+def serve_resource_file(path):
+    dir_path = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', 'frontend/'))
```

Comment:
What we should probably do is make webpack build everything and stick it into a static folder for Flask. I'll adjust the bash script to make that work.